pay-admin/admin/views/corp_type.py

```diff
@@ -0,0 +1,62 @@
+"""Copyright 2021 Province of British Columbia.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from pay_api.models import CorpType, db
+
+from .secured_view import SecuredView
+
+
+class CorpTypeConfig(SecuredView):
+    """Corp Type config."""
+
+    # Allow export as a CSV file.
+    can_export = False
+
+    # Allow the user to change the page size.
+    can_set_page_size = True
+
+    column_display_pk = True
+
+    can_delete = False
+
```

Comment:
does it make sense to move this variables to securedview and give default?i am not sure if it will work that way

Reply:
changes committed